src/DoubleTheDonation/FormExtension/Template/Variations/NeedsSetup.tsx

```diff
@@ -1,13 +1,25 @@
-import {__} from '@wordpress/i18n';
+import {__, sprintf} from '@wordpress/i18n';
+import {Interweave}  from 'interweave';
 import './styles.scss';
 
+declare const window: {
+    GiveDTD: {
+        settingsPage: boolean;
+    };
+} & Window;
+
 /**
  * @unreleased
  */
 export default () => {
     return (
         <div className="give-dtd-block-template">
-            {__('This block requires additional setup. Go to settings to connect your account.', 'give-double-the-donation')}
+            <Interweave
```

Comment:
Is there a reason to use Interweave here?

Reply:
Yes, and that is convenience. By using Interweave I get a clean translatable string in which I can insert any HTML I need.

Reply:
Okay, are you thinking that would come from the server in the future?